tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+    - include: brew-tap.yml
```

Comment:
give meaningful names to files.
add-brew-tap.yml
install-additional-software.yml
install-oh-my-zsh.yml
install-ukrainian-language.yml